tests/test_batch.py

```diff
@@ -61,3 +65,60 @@ def test_create_batch_counter_with_timeout_and_profile(self):
     def test_add_statement(self, batch, statement):
         # just check that does not raise any error
         batch.add_statement(statement)
+
+    @pytest.mark.parametrize(
+        "consistency",
+        [
+            Consistency.ANY,
+            Consistency.ONE,
+            Consistency.TWO,
+            Consistency.THREE,
+            Consistency.QUORUM,
+            Consistency.ALL,
+            Consistency.LOCAL_QUORUM,
+            Consistency.EACH_QUORUM,
+            Consistency.SERIAL,
+            Consistency.LOCAL_SERIAL,
+            Consistency.LOCAL_ONE,
+        ],
+    )
+    def test_set_consistency(self, consistency):
+        batch = create_batch_logged()
+        batch.set_consistency(consistency)
+        batch = create_batch_unlogged()
+        batch.set_consistency(consistency)
+        batch = create_batch_counter()
+        batch.set_consistency(consistency)
+
+    @pytest.mark.parametrize(
+        "serial_consistency",
+        [Consistency.SERIAL, Consistency.LOCAL_SERIAL],
+    )
+    def test_set_serial_consistency(self, serial_consistency):
+        batch = create_batch_logged()
+        batch.set_consistency(serial_consistency)
+        batch = create_batch_unlogged()
+        batch.set_consistency(serial_consistency)
+        batch = create_batch_counter()
+        batch.set_consistency(serial_consistency)
+
+    def test_set_timestamp(self, batch):
+        batch.set_timestamp(time.time())
+        batch.set_timestamp(None)
+
+    def test_set_is_idempotent(self, batch):
+        batch.set_is_idempotent(True)
+        batch.set_is_idempotent(False)
+        batch.set_is_idempotent(None)
+
+    async def test_set_retry_policy(self, batch):
+        batch.set_retry_policy("default")
+        batch.set_retry_policy("fallthrough")
+        batch.set_retry_policy("default", retry_policy_logging=True)
+        batch.set_retry_policy("fallthrough", retry_policy_logging=True)
+        batch.set_retry_policy(None)
+
+    async def test_set_tracing(self, batch):
+        batch.set_tracing(True)
+        batch.set_tracing(False)
+        batch.set_tracing(None)
```

Comment:
hats off!

Reply:
@pfreixes Thanks! What do you think about bump version to 1.0.0 beta?